pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -4,24 +4,46 @@
   buildDotnetModule,
   copyDesktopItems,
   desktop-file-utils,
+  dos2unix,
   dotnetCorePackages,
   fetchFromGitHub,
+  fetchpatch,
+  imagemagick,
   lib,
   runCommand,
+  xdg-utils,
   pname ? "nexusmods-app",
 }:
+let
+  # From https://nexus-mods.github.io/NexusMods.App/developers/Contributing/#for-package-maintainers
+  constants = [
+    # Tell the app it is a distro package; affects wording in update prompts
+    "INSTALLATION_METHOD_PACKAGE_MANAGER"
+
+    # Don't include upstream's 7zz binary; we use the nixpkgs version
+    "NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR"
+  ];
+in
 buildDotnetModule (finalAttrs: {
   inherit pname;
-  version = "0.4.1";
+  version = "0.6.0";
 
   src = fetchFromGitHub {
     owner = "Nexus-Mods";
     repo = "NexusMods.App";
     rev = "v${finalAttrs.version}";
     fetchSubmodules = true;
-    hash = "sha256-FzQphMhiC1g+6qmk/R1v4rq2ldy35NcaWm0RR1UlwLA=";
+    hash = "sha256-8SnHHZSIaX4elQJPi832XGnntsS/+H38724PaTkQLhk=";
   };
 
+  patches = [
+    # Backport fix for NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR
+    (fetchpatch {
+      url = "https://github.com/Nexus-Mods/NexusMods.App/pull/1919.patch";
+      hash = "sha256-5eSOQxvih04yFxf1Uxm1cNDcyQHul8u3yJgrtfa5Gt8=";
+    })
+  ];
```

Comment:
Looks like upstream is prepping to release 0.6.1. If we hold off until it is released we will no longer need this patch.